### DISCUSSION TITLE
JavaFX

### DISCUSSION THREAD

**Discussion by cohe** [microsoft/openjdk#23] [Q&A] [answered]
Once upon a time there was WebStart function part of Java JRE/JDK package as well as all **JavaFX** components.

After using AdoptOpenJDK + IcedTea for web starts + customized compiled JavaFX binaries
there is a new hope in the galaxy to have easier access to up-to-date Java packages containing a JRE capable of Web start and JavaFX applications.

**Q:** Will the Microsoft Java build already contain JavaFX libraries so that it is able to run JavaFX applications?
(or is it required/compatible to add the installation of openJFX)

Best regards and thank you in advance,
cohe

**Comment by karianna** [comment, marked as answer]
@cohe At this stage, we would recommend looking at [OpenWebStart](https://openwebstart.com/) from Karakun and [JavaFX](https://gluonhq.com/products/javafx/) from Gluon.

I'd be interested in hearing more about your use case though!  What sort of applications are you building?
